### ISSUE TITLE
Markdown missing alt for images

### ISSUE THREAD

**Issue by AmelUlMulk** [firecmsco/firecms#673]
How can one add alt tag for images when adding markdown?

**Comment by fgatti675** [comment]
Hi @AmelUlMulk this is not possible at the moment, keeping it as an enhancement request

**Comment by danieljancar** [comment]
@fgatti675 any status update? Thanks 🥇

**Comment by fgatti675** [comment]
Yes this is in FireCMS since a few versions! Just right click on an uploaded image :)
